x-pack/plugins/ml/server/lib/alerts/register_jobs_monitoring_rule_type.ts

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { KibanaRequest } from 'kibana/server';
+import { ML_ALERT_TYPES } from '../../../common/constants/alerts';
+import { PLUGIN_ID } from '../../../common/constants/app';
+import { MINIMUM_FULL_LICENSE } from '../../../common/license';
+import {
+  anomalyDetectionJobsHealthRuleParams,
+  AnomalyDetectionJobsHealthRuleParams,
+} from '../../routes/schemas/alerting_schema';
+import { RegisterAlertParams } from './register_ml_alerts';
+import {
+  ActionGroup,
+  AlertInstanceContext,
+  AlertInstanceState,
+  AlertTypeState,
+} from '../../../../alerting/common';
+
+export type AnomalyDetectionJobsHealthAlertContext = {
+  jobIds: string[];
+  message: string;
+} & AlertInstanceContext;
+
+export const ANOMALY_DETECTION_JOB_REALTIME_ISSUE = 'anomaly_detection_realtime_issue';
+
+export type AnomalyDetectionJobRealtimeIssue = typeof ANOMALY_DETECTION_JOB_REALTIME_ISSUE;
+
+export const REALTIME_ISSUE_DETECTED: ActionGroup<AnomalyDetectionJobRealtimeIssue> = {
+  id: ANOMALY_DETECTION_JOB_REALTIME_ISSUE,
+  name: i18n.translate('xpack.ml.jobsHealthAlertingRule.actionGroupName', {
+    defaultMessage: 'Realtime issue detected',
+  }),
+};
+
+export function registerJobsMonitoringRuleType({
+  alerting,
+  mlServicesProviders,
+  logger,
+}: RegisterAlertParams) {
+  alerting.registerType<
+    AnomalyDetectionJobsHealthRuleParams,
+    AlertTypeState,
+    AlertInstanceState,
+    AnomalyDetectionJobsHealthAlertContext,
+    AnomalyDetectionJobRealtimeIssue
+  >({
+    id: ML_ALERT_TYPES.AD_JOBS_HEALTH,
+    name: i18n.translate('xpack.ml.jobsHealthAlertingRule.name', {
+      defaultMessage: 'Anomaly detection jobs health monitoring',
```

Comment:
"Monitoring" feels a bit like a loaded term and doesn't seem appear in any of the other rule types. However, there is a "uptime monitor status". Could we shorten ours or change it to one of these options?:
> Anomaly detection jobs health
> Anomaly detection health
> Anomaly detection status

Reply:
Changed to `Anomaly detection jobs health` in https://github.com/elastic/kibana/pull/106084/commits/9c0b3ecfa61edfea97ca0881482ca69af5ebd954